### PR TITLE
#106/ TransactionResult type differs from protobuf schema

### LINKED ISSUE #106: TransactionResult type differs from protobuf schema
This is related to: https://github.com/onflow/flow-go-sdk/issues/769

### PR DESCRIPTION
**Pull request by lealobanov**
Closes: #106 

## Description

Adds the missing fields to `FlowTransactionResult`:
- `computationUsage`

As well as missing fields which the Go SDK had but the JVM SDK did not:
- `blockId`
- `blockHeight`
- `collectionId`
- `transactionId`
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-jvm-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels 

## Release Notes

- **New Features**
  - Added new sections in documentation for "Get Execution Data" and "Streaming Events and Execution Data" in both Java and Kotlin examples, enhancing user guidance on retrieving execution data.
  
- **Bug Fixes**
  - Improved error handling in subscription tests for execution data and events to ensure proper exception management.

- **Documentation**
  - Updated README files to include new examples and improved formatting for clarity and consistency.
  - Enhanced the "Examples summary" section in both Java and Kotlin documentation to reflect new entries.
  - Added detailed descriptions and links to relevant code examples for better user understanding.

### CODE REVIEW

File: sdk/src/main/kotlin/org/onflow/flow/sdk/models.kt [hunk 1]
```diff
@@ -267,15 +267,25 @@ data class FlowTransactionResult(
     val status: FlowTransactionStatus,
     val statusCode: Int,
     val errorMessage: String,
-    val events: List<FlowEvent>
+    val events: List<FlowEvent>,
+    val blockId: FlowId,
+    val blockHeight: Long,
+    val transactionId: FlowId,
+    val collectionId: FlowId,
+    val computationUsage: Long
```
[the remark this answers is not in the document]
Comment (franklywatson) [reply sdk/src/main/kotlin/org/onflow/flow/sdk/models.kt:275]: @lealobanov what do you think of this suggestion? Seems maybe reasonable?

**Review by franklywatson — APPROVED** [review]
Looks good to me, but would request that @jribbink take a look to make sure before merging.

**Review by jribbink — APPROVED** [review]
Looks good to me, I would also suggest to make the change @franklywatson suggested before merging

File: sdk/src/main/kotlin/org/onflow/flow/sdk/models.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (lealobanov) [reply sdk/src/main/kotlin/org/onflow/flow/sdk/models.kt:275]: Yes will do, thanks!

[the remark this answers is not in the document]
Comment (lealobanov) [reply sdk/src/main/kotlin/org/onflow/flow/sdk/models.kt:275]: This has been updated

**Pull request merged**